### PR TITLE
[Github Actions] Rename docker secrets and parameterize docker user

### PR DESCRIPTION
**Pull request by evantahler**
Related to https://github.com/airbytehq/airbyte-cloud/issues/1763

This PR renames the secrets so that old branches stop working and folks will need to rebase from master to publish from here on out.  I also require a secret for the docker username.

Note: DO NOT update the secrets until this PR is merged to master!

### CODE REVIEW

File: tools/bin/gke-kube-acceptance-test/acceptance_test_kube_gke.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ function findAndDeleteTag () {
 }
 
 function cleanUpImages () {
-    TOKEN=$(curl --request POST 'https://hub.docker.com/v2/users/login/' --header 'Content-Type: application/json' --data-raw '{"username":"airbytebot","password":"'$DOCKER_PASSWORD'"}' | jq '.token')
+    TOKEN=$(curl --request POST 'https://hub.docker.com/v2/users/login/' --header 'Content-Type: application/json' --data-raw '{"username":"'$DOCKER_HUB_USERNAME'","password":"'$DOCKER_HUB_PASSWORD'"}' | jq '.token')
```
Comment (pedroslopez) [inline tools/bin/gke-kube-acceptance-test/acceptance_test_kube_gke.sh:47]: hmm - this one is also grabbing a token through curl, do we need to `set +x` `set -x` here as well?

File: tools/integrations/manage.sh [hunk 2]
```diff
@@ -213,9 +213,18 @@ cmd_publish() {
   docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 
   # log into docker
-  DOCKER_USERNAME=${DOCKER_USERNAME:-airbytebot}
+  if test -z "${DOCKER_HUB_USERNAME}"; then
+    echo 'DOCKER_HUB_USERNNAME not set.';
```
Comment (pedroslopez) [inline tools/integrations/manage.sh:217]:
```suggestion
    echo 'DOCKER_HUB_USERNAME not set.';
```

**Review by pedroslopez — COMMENTED** [review]
Looks good, one question on token generation for `acceptance_test_kube_gke`

**Review by cgardens — APPROVED** [review]

File: tools/bin/release_version.sh [hunk 3]
```diff
@@ -14,12 +14,17 @@ if [[ -z "${CLOUDREPO_PASSWORD}" ]]; then
   exit 1;
 fi
 
-if [[ -z "${DOCKER_PASSWORD}" ]]; then
-  echo 'DOCKER_PASSWORD for airbytebot not set.';
+if [[ -z "${DOCKER_HUB_USERNAME}" ]]; then
+  echo 'DOCKER_HUB_USENAME not set.';
```
Comment (pedroslopez) [inline tools/bin/release_version.sh:18]:
```suggestion
  echo 'DOCKER_HUB_USERNAME not set.';
```

File: tools/bin/gke-kube-acceptance-test/acceptance_test_kube_gke.sh (hunk 1 above)
Comment (evantahler) [reply tools/bin/gke-kube-acceptance-test/acceptance_test_kube_gke.sh:47]: I don't think so, because no where in the file do we turn that on (set +x)?

Comment (evantahler) [reply tools/bin/gke-kube-acceptance-test/acceptance_test_kube_gke.sh:47]: ... but maybe we should?

Comment (pedroslopez) [reply tools/bin/gke-kube-acceptance-test/acceptance_test_kube_gke.sh:47]: Ah! I see, it was specifically enabled on `manage.sh` - should be ok then

**Review by pedroslopez — APPROVED** [review]

**Pull request merged**